frontend/src/pages/spotnet/dashboard/Dashboard.jsx

```diff
@@ -42,22 +42,22 @@ export default function Component({ telegramId }) {
     {
       title: 'Collateral & Earnings',
       icon: CollateralIcon,
-      balance: '0.00',
+      balance: '0.039404186081257303',
       currencyName: 'Ethereum',
       currencyIcon: EthIcon,
     },
     {
       title: 'Borrow',
       icon: BorrowIcon,
-      balance: '0.0',
+      balance: '-55.832665',
       currencyName: 'USD Coin',
       currencyIcon: UsdIcon,
     },
   ]);
 
-  const [healthFactor, setHealthFactor] = useState('0.00');
-  const [startSum, setStartSum] = useState(0);
-  const [currentSum, setCurrentSum] = useState(0);
+  const [healthFactor, setHealthFactor] = useState('1.47570678');
+  const [startSum, setStartSum] = useState(200);
+  const [currentSum, setCurrentSum] = useState(200);
```

Comment:
don't change this.

Reply:
should I also change this value to the original values?

Reply:
you shouldn't have changed any values